### DISCUSSION TITLE
Disable typing

### DISCUSSION THREAD

**Discussion by prousso** [sqlalchemy/sqlalchemy#11612] [Usage Questions]
Hi, is there a way to disable typing in sqla2.0
We had never typed (with mypy plugin) before and the gap is too huge to do in one PR (requirements bump).
Thank you

**Comment by CaselIT** [comment]
Hi,

Not sure what you mean by disable.
What issue are you seeing that make you want didable it?

**Reply by prousso** [reply]
We recently migrated all our code from query API to select API to have a code base compatible with 2.0 (code run in production)
But mypy detects so many errors that I can't fix it quickly. At the same time, I want to bump our dependency to 2.0 and I want to know if there is a possibility to disable static typing added by 2.0

**Reply by zzzeek** [reply]
follow mypy's instructions here: https://mypy.readthedocs.io/en/stable/existing_code.html

**Reply by prousso** [reply]
I already follow these and 
```toml
[[tool.mypy.overrides]]
module = ["sqlalchemy.*"]
ignore_errors = true
```
doesn't work

**Reply by CaselIT** [reply]
That's not something a library can do. You should look into mypy options like mike suggested.
